circle.yml

```diff
@@ -59,7 +59,8 @@ jobs:
           name: Install Docker Compose
           command: |
             set -x
-            sudo pip install docker-compose
+            pip install docker-compose>=1.18
```

Comment:
future improvement-idea, move this to a separate `circleci-uitests.txt` in `requirements/` so that it can be tracked by pyup.